tests/Functional/ContentNewsRenderTest.php

```diff
@@ -70,6 +76,70 @@ public function testNewsRendering(): void {
     $node->save();
     $this->drupalGet($node->toUrl());
 
+    // Assert news contacts.
+    $contacts = $this->assertSession()->elementExists('css', 'div#news-contacts');
+
+    $contact_headers = $contacts->findAll('css', 'h2');
+    $this->assertEquals('Contacts', $contact_headers[0]->getText());
+
+    $contact_name = $contacts->findAll('css', 'h3');
+    $this->assertCount(1, $contact_name);
+    $this->assertEquals('general_contact', $contact_name[0]->getText());
+
+    $contact_body = $contacts->findAll('css', '.ecl-editor');
+    $this->assertCount(1, $contact_body);
+    $this->assertEquals('Body text general_contact', $contact_body[0]->getText());
+
+    $contacts_html = $contacts->getHtml();
+    $field_list_assert = new FieldListAssert();
+    $contact_expected_values = [
+      'items' => [
+        [
+          'label' => 'Organisation',
+          'body' => 'Organisation general_contact',
+        ], [
+          'label' => 'Website',
+          'body' => 'http://www.example.com/website_general_contact',
+        ], [
+          'label' => 'Email',
+          'body' => 'general_contact@example.com',
+        ], [
+          'label' => 'Phone number',
+          'body' => 'Phone number general_contact',
+        ], [
+          'label' => 'Mobile number',
+          'body' => 'Mobile number general_contact',
+        ], [
+          'label' => 'Fax number',
+          'body' => 'Fax number general_contact',
+        ], [
+          'label' => 'Postal address',
+          'body' => 'Address general_contact, 1001 Brussels, Belgium',
+        ], [
+          'label' => 'Office',
+          'body' => 'Office general_contact',
+        ], [
+          'label' => 'Social media',
+          'body' => html_entity_decode('&nbsp;') . 'Social media general_contact',
```

Comment:
Might be we should fix template instead of using `html_entity_decode('&nbsp;')`?

Reply:
The ECL component uses `&nbsp;` for the layout, check: https://ec.europa.eu/component-library/ec/components/navigation/link/code/

If we remove it we get a different layout.
Without `&nbsp;`:
<img width="310" alt="link_no_space" src="https://user-images.githubusercontent.com/5982573/94415116-73c8a080-017d-11eb-93c7-85b8ddafb6be.png">

With `&nbsp;`:
<img width="330" alt="link_space" src="https://user-images.githubusercontent.com/5982573/94415121-74613700-017d-11eb-89b2-e1e931913b73.png">

Reply:
Clear. Thank you. Let's leave it as it.